airbyte-webapp/src/components/connection/ConnectionInfoCard/ConnectionInfoCard.tsx

```diff
@@ -65,9 +63,9 @@ export const ConnectionInfoCard: React.FC = () => {
         >
           <ConnectorCard
             connectionName={destination.destinationName}
-            icon={destinationDefinition?.icon}
+            icon={destDefinition?.icon}
```

Comment:
From the comment in the [issue](https://github.com/airbytehq/airbyte/issues/21661): it should be using the icon source or destination  directly instead of the definition.

Reply:
Oh, thanks for pointing me out 👍 Fixed